doc/nrf/device_guides/nrf91/nrf91_cloud_certificate.rst

```diff
@@ -99,9 +99,9 @@ Complete the following steps to provision the certificate:
 
    .. tabs::
 
-      .. group-tab:: nRF9161 DK
+      .. group-tab:: nRF91x1 DK
 
-         .. figure:: images/cellularmonitor_selectdevice_nrf9161.png
+         .. figure:: images/cellularmonitor_selectdevice_nrf9151.png
             :alt: Cellular Monitor - Select device
 
             Cellular Monitor - Select device
```

Comment:
Maybe:
```suggestion
            :alt: Cellular Monitor - Select device (nRF9151 DK shown)

            Cellular Monitor - Select device (nRF9151 DK shown)
```
So that we can have 1 image for 9161 and 9151? Global edit for similar images.